lib/HLSL/DxilContainerReflection.cpp

```diff
@@ -28,8 +28,13 @@
 
 #include "dxc/dxcapi.h"
 
+#ifdef LLVM_ON_WIN32
 #include "d3d12shader.h" // for compatibility
 #include "d3d11shader.h" // for compatibility
+#endif
+
+#ifdef LLVM_ON_WIN32
```

Comment:
Maybe merge this with the previous #define?

Reply:
oh yes! Done.